### PR TITLE
refactor/replace tailwind with styled components

### PR DESCRIPTION
**Pull request by valfaro123**
### 🔥 Summary
-Remove tailwind and replace with styled components to fit the rest of the app

### 😤 Problem / Goals
-We have started the transition from using tailwind to styled components
-The app currently has some components that need to be converted 

### 🤓 Solution
-Replace old tailwind classes with styled components

### CODE REVIEW

File: frontend/src/components/layout/PageWrapper/PageWrapper.tsx [hunk 1]
```diff
@@ -58,3 +58,28 @@ export const PageWrapper: React.FC<PageWrapperProps> = ({
...
+const ContentWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  padding: 0 ${pxToRem(24)};
+
+  @media (min-width: 768px) {
+    padding: 0 ${pxToRem(32)};
+  }
+  @media (min-width: 1280px) {
+    padding: 0 ${pxToRem(48)};
```
Comment (Thistle-Strum) [inline frontend/src/components/layout/PageWrapper/PageWrapper.tsx:79]: These breakpoint pixel values can be replaced with the breakpoints obj in styled-theme.ts

File: frontend/src/components/tables/LatestBlocks/LatestBlocks.tsx [hunk 2]
```diff
@@ -115,3 +116,34 @@ export const LatestBlocks: React.FC<LatestBlocksProps> = ({
...
+  padding: ${pxToRem(20)};
+`;
+
+const FooterContent = styled(NavLink)`
+  background-color: ${colors.lightGrey};
+  color: ${colors.darkRed};
+  padding: 0.5rem 1rem;
+  width: fit-content;
+  border-radius: 0.375rem;
+  font-weight: 500;
```
Comment (Thistle-Strum) [inline frontend/src/components/tables/LatestBlocks/LatestBlocks.tsx]: Font weights are also in styled-theme

File: frontend/src/components/tables/BlockTable/BlockTable.tsx [hunk 3]
```diff
@@ -140,3 +141,33 @@ export const BlockTable: React.FC<BlockTableProps> = ({
...
+
+const ShowMoreButton = styled.button`
+  background-color: ${colors.lightGrey};
+  color: ${colors.darkRed};
+  min-width: ${pxToRem(150)};
+  padding: 0.5rem 0;
+  width: fit-content;
+  border-radius: 0.375rem;
+  border: none;
+  font-weight: 500;
```
Comment (Thistle-Strum) [inline frontend/src/components/tables/BlockTable/BlockTable.tsx]: `${fontWeight.medium}`

File: frontend/src/components/tables/BlockTable/BlockTable.tsx [hunk 4]
```diff
@@ -140,3 +141,33 @@ export const BlockTable: React.FC<BlockTableProps> = ({
...
+
+  :hover {
+    background-color: ${colors.lightRed};
+  }
+`;
+const BlockTableTitle = styled.p`
+  font-weight: bold;
```
Comment (Thistle-Strum) [inline frontend/src/components/tables/BlockTable/BlockTable.tsx]: `${fontWeight.bold}`

File: frontend/src/components/tables/LatestBlocks/LatestBlocks.tsx [hunk 5]
```diff
@@ -115,3 +116,34 @@ export const LatestBlocks: React.FC<LatestBlocksProps> = ({
...
+
+  :hover {
+    background-color: ${colors.lightRed};
+  }
+`;
+
+const BlockTableTitle = styled.p`
+  font-weight: bold;
```
Comment (Thistle-Strum) [inline frontend/src/components/tables/LatestBlocks/LatestBlocks.tsx]: `${fontWeight.bold}`

File: frontend/src/components/tables/PeerTable/PeerTable.tsx [hunk 6]
```diff
@@ -42,3 +47,22 @@ export const PeerTable: React.FC<PeerTableProps> = ({ peers }) => {
...
+`;
+
+const PeerTableHead = styled.div`
+  display: flex;
+  padding-left: 2rem;
+`;
+
+const HeadLabel = styled.p`
+  color: black;
+  font-weight: bold;
```
Comment (Thistle-Strum) [inline frontend/src/components/tables/PeerTable/PeerTable.tsx]: `${fontWeight.bold}`

**Review by Thistle-Strum — APPROVED** [review]

**Review by hoffmannjan — CHANGES_REQUESTED** [review]
After removing all of the tailwind imports / dependencies a few things broke - here are screenshots:
Screenshot 2023-01-03 at 16 02 57

screenshot_2023-01-03_at_16 02 33

so please go through the views and make sure all is complete :) 

other then this LGTM!

**Review by hoffmannjan — APPROVED** [review]
OK I helped with conflicts and updated few things, looking good, only problem is that it now fails on the tests, @Thistle-Strum could you look at this?

**Pull request merged**